go/vt/binlog/binlog_streamer_rbr_test.go

```diff
@@ -519,17 +521,12 @@ func TestStreamerParseRBRNameEscapes(t *testing.T) {
 
 	go sendTestEvents(events, input)
 	_, err := bls.parseEvents(context.Background(), events, errs)
-	if err != ErrServerEOF {
-		t.Errorf("unexpected error: %v", err)
-	}
+	require.Equal(t, ErrServerEOF, err, "unexpected error: %v", err)
 
-	if !reflect.DeepEqual(got, want) {
-		t.Errorf("binlogConnStreamer.parseEvents(): got:\n%+v\nwant:\n%+v", got, want)
-		for i, fbt := range got {
-			t.Errorf("Got (%v)=%v", i, fbt.statements)
-		}
-		for i, fbt := range want {
-			t.Errorf("Want(%v)=%v", i, fbt.statements)
-		}
+	assert.True(t, reflect.DeepEqual(got, want), "binlogConnStreamer.parseEvents(): got:\n%+v\nwant:\n%+v", got, want)
+
+	for i, fbt := range got {
+		assert.Equal(t, fbt.statements, want[i].statements, "Got (%v)=%v, want (%v)=%v", i, fbt.statements, i, want[i].statements)
```

Comment:
This seems superfluous given we've validated the equality of the entire slice, above?